projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/sentence/entity/Court.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.sentence.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import org.hibernate.annotations.Immutable
+
+@Entity
+@Immutable
+class Court(
+
+    @Id
+    @Column(name = "court_id")
+    val id: Long,
+
+    @Column(name = "court_name")
```

Comment:
looks like court_name is nullable on DB, would cause issues on deployment. Make name optional

Reply:
i copied the structure from another project, and have checked the same class in other projects.  however, to keep it consistent with the db, making it a nullable field.